rllib/algorithms/marwil/marwil.py

```diff
@@ -361,6 +362,9 @@ def build_learner_connector(
         pipeline.append(
             GeneralAdvantageEstimation(gamma=self.gamma, lambda_=self.lambda_)
         )
+        pipeline.append(
```

Comment:
Makes sense. GAE connector outputs tensors already due to it requiring a VF forward pass (with the tensors coming from `NumpyToTensor`). It does get a little more complicated now in the pipeline, but I feel like it's still ok (not too crazy, connector pieces are named properly, each piece performs a well distinguished task, ...).

Reply:
Yes. And keep in mind: The data workers will run in parallel and prefetch batches which will actually make the pipeline quite smooth. Another connector piece or one less will not make a big difference, if at all. User usually have enough resources to run multiple data workers in parallel and they should.